internal/configs/virtualserver.go

```diff
@@ -14,6 +14,16 @@ import (
 
 const nginx502Server = "unix:/var/run/nginx-502-server.sock"
 
+var incompatibleLBMethodsForSlowStart = map[string]bool{
+	"random":                          true,
+	"ip_hash":                         true,
+	"hash":                            true,
```

Comment:
hash is not required here, considering this change https://github.com/nginxinc/kubernetes-ingress/pull/655/files#diff-d8f2204ffe0ef11d167489c09e9f4170R380